tests/fs-test.rs

```diff
@@ -71,20 +71,19 @@ fn get_testname_derived_guest_path(test_name: &str) -> PathBuf {
 
 // Creates a vector out of a given host path and guest path for UhyveFileMap.
 fn create_filemap_params<T: AsStr, U: AsStr>(host_path: T, guest_path: U) -> Vec<String> {
-	[format!("{}:{}", host_path.as_str(), guest_path.as_str())].to_vec()
+	vec![format!("{}:{}", host_path.as_str(), guest_path.as_str())]
 }
 
 /// Creates kernel arguments for fs-tests.
 ///
 /// * `test_name` - Name of the test.
 /// * `file_path` - This parameter defines the guest path that the kernel will open.
 fn create_kernel_args<T: AsStr, U: AsStr>(test_name: T, file_path: U) -> Vec<String> {
-	[
+	vec![
 		("--".to_owned()),
```

Comment:
I think the parentheses around these list items are also unnecessary.

Reply:
nice catch